src/util/teleport.hpp

```diff
@@ -83,12 +84,13 @@ namespace YimMenu::Teleport
 		return true;
 	}
 
-	inline Vector3 GetWaypointCoords()
+	inline Vector3 GetWaypointCoords(bool showNotification = false)
```

Comment:
Instead of adding a param here, you can just use the native `IS_WAYPOINT_ACTIVE` inside the looped command. Changing util function definitions according to commands is bad practice. Not that this would break anything, just as a rule of thumb.